db/types.go

```diff
@@ -62,6 +62,9 @@ type Job struct {
 
 	// Output list of the given job
 	Outputs []TranscodeOutput `redis-hash:"-" json:"outputs"`
+
+	// AudioDownmix holds source and output channels for configuring downmixing
+	AudioDownmix AudioDownmix `json:"audioDownmix"`
```

Comment:
I think we can avoid having to define an `IsSet()` function if we make `AudioDownmix` a pointer and add `omitempty` to the json tag

```suggestion
AudioDownmix *AudioDownmix `json:"audioDownmix,omitempty"`